core_test/carmirror_test.go

```diff
@@ -251,7 +251,7 @@ func MockBatchTransfer(sender_store *mock.Store, receiver_store *mock.Store, roo
 
 	go func() {
 		log.Debugf("timeout started")
-		time.Sleep(5 * time.Second)
+		time.Sleep(10 * time.Second)
```

Comment:
Again OOC, was this timer bump a fix of some kind, or just left over after debugging?

Reply:
It's kind of left over from debugging, but I think we should let it stand as we are likely to have tests that take longer as we add serialization and deserialization to the mock transfer.